rust/src/lib/nispor/apply.rs

```diff
@@ -44,6 +45,10 @@ pub(crate) fn nispor_apply(
     let mut net_conf = nispor::NetConf::default();
     net_conf.ifaces = Some(np_ifaces);
 
+    if merged_state.routes.is_changed() {
+        net_conf.routes = Some(gen_nispor_route_confs(&merged_state.routes)?);
+    }
+
```

Comment:
We are using Nispor to apply the static route, not sure if this is deviating from the purpose of querying Linux network state for Nispor ?